core/storage/user/gae_models.py

```diff
@@ -43,6 +43,12 @@ class UserSettingsModel(base_models.BaseModel):
     # When the user last logged in. This may be out-of-date by up to
     # feconf.PROXIMAL_TIMEDELTA_SECS seconds.
     last_logged_in = ndb.DateTimeProperty(default=None)
+    # When the user last edited an exploration in. This may be out-of-date by up
```

Comment:
I think these two new fields should be exact (and not be subject to the PROXIMAL_TIMEDELTA_SECS error margin). We had a margin of error for last_logged_in because it's such a frequent occurrence to ping a handler while logged in, but these actions occur much less frequently.

Also I would call them last_edited_an_exploration and last_created_an_exploration, for parity with the other last_* fields.